index.bs

```diff
@@ -4508,7 +4508,7 @@ closed. It is only used internally, and is never exposed to web developers.
  The <dfn id="ws-default-controller-signal" attribute
  for="WritableStreamDefaultController">signal</dfn> getter steps are:
 
- 1. Return [=this=].[=WritableStreamDefaultController/[[signal]]=].
+ 1. Return [=this=].[=WritableStreamDefaultController/[[abortController]]=].{{AbortController/signal}}.
```

Comment:
You need to return the value of the "internal slot" and not reference the "public" getter.

Reply:
If I understand [Bikeshed's autolink syntax](https://speced.github.io/bikeshed/#autolink-wrappers) correctly, this should do the trick:
```suggestion
 1. Return [=this=].[=WritableStreamDefaultController/[[abortController]]=].[=AbortController/signal=].
```

Reply:
It'd have to be something like
```suggestion
 1. Return [=this=].[=WritableStreamDefaultController/[[abortController]]=]'s [=AbortController/signal=].
```
as we don't define dot syntax for these.